app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -615,9 +616,14 @@ public void someResultsFound() {
      */
     private void buildDepictionList(List<IdAndCaptions> idAndCaptions) {
         depictionContainer.removeAllViews();
+        String locale = Locale.getDefault().getLanguage();
         for (IdAndCaptions idAndCaption : idAndCaptions) {
                 depictionContainer.addView(buildDepictLabel(
-                    idAndCaption.getCaptions().values().iterator().next(),
+                    //Check if the Depiction Caption is available in user's locale if not then check for english, else show any available.
```

Comment:
I personally love ternary operations but using 2 levels of ternary makes it hard to read like a very long sentence. I am good to merge this PR after you turned them to use if braces. :)

Reply:
Done!